doc/requirements.txt

```diff
@@ -40,3 +40,4 @@ pyDOE2
 graphviz
 
 fake_useragent
+currencyconverter
```

Comment:
Do we need the `currencyconverter `also here? The idea is to use this specification to generate an environment needed only for generating the documentation locally. [Agree though that it's not the most consistent approach]